mobile/native/src/api.rs

```diff
@@ -328,6 +328,12 @@ pub async fn register_beta(email: String) -> Result<()> {
     order::register_beta(email).await
 }
 
+/// Send the Firebase token to the LSP for push notifications
+#[tokio::main(flavor = "current_thread")]
+pub async fn update_fcm_token(fcm_token: String) -> Result<()> {
+    order::update_fcm_token(fcm_token).await
```

Comment:
🔧 the orders module does not seem to be the right one for the `update_fcm_token` function. How about creating a `users` module?

Reply:
good point. moved to `users` module, along with registration part, which didn't fit there either 😊